### PR TITLE
New Lily ideas

### PR DESCRIPTION
**Pull request by lilygruhl**

### CODE REVIEW

File: ideas.html [hunk 1]
```diff
@@ -1 +1,5 @@
-
+<h1>Lily idea #1</h1>
+<p>
+  You create your sandwich by draging items into the sandwich anyway you want. Then you place your order and pay.
+  then it asks you if you wnat to play sandwich games. If you get the high score you get 10% of the next sandwich you buy!
```
Comment (jgruhl) [inline ideas.html:13]: How do we store the high score?

File: ideas.html [hunk 2]
```diff
@@ -1 +1,5 @@
-
+<h1>Lily idea #1</h1>
```
Comment (jgruhl) [inline ideas.html]: I like the idea keep them coming!!!

File: ideas.html (hunk 1 above)
Comment (jasper-gruhl) [reply ideas.html:13]: Ya how? I have no clue.

Comment (lilygruhl) [reply ideas.html:13]: IDK.

Comment (lilygruhl) [reply ideas.html:13]: Maybe we could look at some other games codes and see how they did that. But like Jasper I have no clue how to do that!

Comment (jgruhl) [reply ideas.html:13]: We'll have to use Javascript and then have our own server to store the information

Comment (lilygruhl) [reply ideas.html:13]: I didn't understand anything you just said

Comment (lilygruhl) [reply ideas.html:13]: I mean I know what Javascript is

File: ideas.html (hunk 2 above)
Comment (lilygruhl) [reply ideas.html]: 🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗

**Pull request merged**